relay-server/src/services/project_cache.rs

```diff
@@ -480,6 +482,7 @@ impl Services {
         test_store: Addr<TestStore>,
         upstream_relay: Addr<UpstreamRelay>,
         global_config: Addr<GlobalConfigManager>,
+        metric_stats: MetricStats,
```

Comment:
Not really happy about this, since metric stats it's not accessed as a service. I could put the param top level.

Reply:
Not a fan either, maybe should just pass it as a separate arg like `config` to be consistent?

Reply:
Yes, lets do that.